backend/tests/daily/embedding/test_embeddings.py

```diff
@@ -108,7 +108,7 @@ def azure_embedding_model() -> EmbeddingModel:
     return EmbeddingModel(
         server_host="localhost",
         server_port=9000,
-        model_name="text-embedding-3-large",
+        model_name="text-embedding-3-small",
```

Comment:
What's the purpose of this change?

Also, the screenshot in the PR description still says "text-embedding-3-large". Not sure if the screenshot was taken prior to your changes...